components/compose/awesomebar/src/main/java/mozilla/components/compose/browser/awesomebar/AwesomeBar.kt

```diff
@@ -37,7 +38,8 @@ fun AwesomeBar(
     onSuggestionClicked: (AwesomeBar.Suggestion) -> Unit,
     onAutoComplete: (AwesomeBar.Suggestion) -> Unit,
     onScroll: () -> Unit = {},
-    profiler: Profiler? = null
+    profiler: Profiler? = null,
+    showEditSuggestions: () -> Boolean = { true },
```

Comment:
I chose to use a function here so we can update this dynamically when we're ready with unified search.

Reply:
Can you elaborate why a function makes sense here? I don't see why this should be decided dynamically each time Suggestions are drawn. In other words, why would this change between queries of the same `AwesomeBar` instance?

Reply:
I could move it up higher.  But I was thinking about future requirement for unified search. When the user switch over the history search, we should dynamically stop showing the button.

Reply:
What do you mean by "When the user switch over the history search"? Happy to talk through next week, the must be a requirement I am missing :).

Reply:
Sure, let's talk.  It could be that I don't need to pass the function down to the `Suggestions` level.  Thanks